src/STW.SubmissionApi/Controllers/VersionController.cs

```diff
@@ -0,0 +1,14 @@
+namespace src.Controllers;
+
+using Microsoft.AspNetCore.Mvc;
+
+[ApiController]
+[Route("/admin/version")]
+public class VersionController : ControllerBase
+{
+    [HttpGet]
+    public string Index()
```

Comment:
We should start a test project `STW.SubmmissionApi.UnitTests` with a test for this endpoint. See the processing API for example https://github.com/DEFRA/stw-processing-api/pull/2